pipeline-templates/terraform-apply.yaml

```diff
@@ -17,10 +18,15 @@ jobs:
               inputs:
                 sourceFolder: '$(Pipeline.Workspace)/${{ parameters.zone }}${{ parameters.build }}plan'
                 targetFolder: '$(Pipeline.Workspace)/drop/components/${{ parameters.zone }}'
-            
-            - task: TerraformInstaller@0
+            - task: Bash@3
+              displayName: Terraform install (tfenv)
               inputs:
-                terraformVersion: ${{ parameters.terraformVersion }}
+                filePath: $(System.DefaultWorkingDirectory)/cnp-azuredevops-libraries/scripts/tfenv-install-terraform.sh
+                workingDirectory: $(System.DefaultWorkingDirectory)
+            - checkout: self
+              clean: true
+              fetchDepth: 5
+              lfs: true
```

Comment:
what made you add these? I wouldn't expect you to need any of them and definitely not lfs =/